src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,47 @@
-import React from 'react';
 import './CurrentUser.scss';
+import React, { useState, useEffect } from 'react';
+import { getUser } from '../../api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+type Props = {
+  userId: number;
+  selectUser: (id: number) => void;
+};
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+export const CurrentUser: React.FC<Props> = ({ userId, selectUser }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [error, setError] = useState('');
+
+  useEffect(() => {
+    getUser(userId)
+      .then(userFromServer => {
+        setUser(userFromServer);
+      })
+      .catch(() => {
+        setError('User is not exist');
+      });
+  }, [userId]);
+
+  return (
+    <>
+      {user ? (
```

Comment:
you have a lot of code in ternary operator, in this case the best way to do it like this
```suggestion
  if (!user) {
    return <span>{error}</span>
  }
  
  return (
    <>
```